compiler/rustc_typeck/src/errors.rs

```diff
@@ -245,8 +245,6 @@ pub struct ExplicitGenericArgsWithImplTrait {
     #[primary_span]
     #[label]
     pub spans: Vec<Span>,
+    #[help]
+    pub is_nightly_build: Option<()>,
```

Comment:
I'm wondering if it could be useful to require the field name in the fluent file for spanless helps. Or generally for optional fields. It could be surprising to see some of them if you look at the entire fluent spec for an error, because you'd see all fields and they'd look "equal".

But considering I have zero fluent experience, that's just me guessing from the files.

Reply:
Maybe, I don't think it's worth changing this right now but it should be fairly easy to change down the line if we want to.